src/plays/dad-jokes/DadJokes.js

```diff
@@ -0,0 +1,32 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import { useState } from 'react';
+import JokeList from './components/jokeList/jokeList';
+import { MdDarkMode, MdLightMode } from 'react-icons/md';
+import './styles.css';
+
+// WARNING: Do not change the entry componenet name
+function DadJokes(props) {
+  const [darkTheme, setDarkTheme] = useState(false);
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```

Comment:
Add a unique class name at this level. We will use that class name as a prefix to your styles in CSS

Reply:
Done